ui/src/index.tsx

```diff
@@ -61,7 +61,14 @@ const initStores = (): StoreMapping => {
 
     registerReactions(stores);
 
-    stores.currentUser.tryAuthenticate().catch(() => {});
+    stores.currentUser.tryAuthenticate().then(() => {
+        // always request notification permission when logged in
+        Notification.requestPermission()
+            .then(perm => console.log("Notification permissions " + perm))
+            .catch(console.error)
+        // reregister worker
+        registerNotificationWorker(stores.currentUser.token());
+    }).catch(() => {});
```

Comment:
Please add another button under the [enable notifications](https://github.com/gotify/server/blob/master/ui/src/layout/Navigation.tsx#L101) button, something like "enable background notifications". Should probably only visible if the notification permission was given. Gotify must not request this permission without user interaction, see https://github.com/gotify/server/issues/264

The button should be a toggle, enabling and disabling the notifications.